README.md

```diff
@@ -49,6 +50,31 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### macOS
+
+ddev Cypress Setup (Mac)
+
+```bash
+# Install XQuartz
+brew install xquartz --cask
+
+# Run XQuartz
+open -a Xquartz
+```
+In the XQuartz preferences, go to the “Security” tab and make sure the “Allow connections from network clients” checkbox is checked
+```bash
+# Run the below command
+xhost + 127.0.0.1
+```
+Add a file called `docker-compose.cypress_extra.yaml` with the following content: 
```

Comment:
File needs to be added to `.ddev/`-folder, would be good to state this explicitly imho